catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -25,9 +25,9 @@
             name="Maximum Concurrent Files" id="numThreads" required="true"
             type="Integer" default="1"/>
 
-        <AD description="Specifies the time to wait (in milliseconds) before acquiring a lock on a file in the monitored directory.  This interval is used for sleeping between attempts to acquire the read lock on a file to be ingested.  The default value of 100 milliseconds is recommended. If the value provided is less than 100, the default value will be used."
+        <AD description="Specifies the time to wait (in milliseconds) before acquiring a lock on a file in the monitored directory.  This interval is used for sleeping between attempts to acquire the read lock on a file to be ingested.  The default value of 500 milliseconds is recommended. If the value provided is less than 100, the default value will be used."
```

Comment:
Yes, the absolute minimum for our CDM is still 100ms.  For slower machines, the old default was too fast (based on user feedback), but faster machines should still be able to handle 100ms without issue.